web/config/wsgi.py

```diff
@@ -29,6 +29,6 @@ def retrying_send(message, *args, **kwargs):
         return _orig_send(message, *args, **kwargs)
 
 
-EmailMessage.send = retrying_send
+EmailMessage.send = retrying_send  # type: ignore  # Where is this being used?
```

Comment:
Currently, not ignoring it will run into this error:
```
config/wsgi.py:32: error: Cannot assign to a method
config/wsgi.py:32: error: Incompatible types in assignment (expression has type "Callable[[Any, VarArg(Any), KwArg(Any)], Any]", variable has type "Callable[[EmailMessage, bool], int]")
```

Reply:
Got it: mypy doesn't like monkey patching https://github.com/python/mypy/issues/2427

I'm going to add a comment to that effect.